core/src/utils/statistics/thread_statistics.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <atomic>
+#include <chrono>
+#include <cstdint>
+#ifndef NDEBUG
+#include <optional>
+#include <thread>
+#endif
+
+#include <userver/utils/datetime/steady_coarse_clock.hpp>
+
+USERVER_NAMESPACE_BEGIN
+
+namespace utils::statistics {
+
+namespace impl {
+
+struct ThreadCpuUsage final {
+  std::chrono::microseconds user{};
```

Comment:
some comment that it is an accumulated value / rusage is needed

Reply:
done